lego/apps/users/models.py

```diff
@@ -293,6 +293,20 @@ def email_address(self):
             return internal_address
         return self.email
 
+    @property
+    def unanswered_surveys(self):
+        # TODO: optimize and test properly
+        from lego.apps.surveys.models import Survey
+        from lego.apps.events.models import Registration
+        registrations = Registration.objects.filter(
+            user_id=self.id, presence=event_constants.PRESENT
+        )
+        unanswered_surveys = Survey.objects.filter(
+            event__registrations__in=registrations, active_from__lte=timezone.now(),
+            template_type__isnull=True
+        ).prefetch_related('event__registrations')
+        return list(unanswered_surveys.values_list('id', flat=True))
```

Comment:
This should be a bit better @finnss. I suppose it is possible to handle this even better with more complex queries, but better than nothing 😊